lntest/itest/lnd_test.go

```diff
@@ -7598,6 +7598,350 @@ func testRevokedCloseRetributionRemoteHodl(net *lntest.NetworkHarness,
 	assertNodeNumChannels(t, dave, 0)
 }
 
+// testRevokedCloseRetributionAltruistWatchtower tests that Dave is able
+// carry out retribution in the event that she fails in state where the remote
+// commitment output has zero-value.
+func testRevokedCloseRetributionAltruistWatchtower(net *lntest.NetworkHarness,
```

Comment:
Needs an updated `godoc` comment, this is copy-pasta from `testRevokedCloseRetributionZeroValueRemoteOutput`.

Reply:
lol you're right, updated godoc